contracts/interfaces/ITimeLock.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { ITimeLock } from '../interfaces/ITimeLock.sol';
+
+interface ITimeLock {
+    error InvalidTimeLockHash();
+    error TimeLockAlreadyScheduled();
+    error TimeLockNotReady();
+
+    function MINIMUM_TIME_LOCK_DELAY() external view returns (uint256);
```

Comment:
```suggestion
    function minimum_time_lock_delay() external view returns (uint256);
```
Make `MINIMUM_TIME_LOCK_DELAY` internal instead and manually create a getter for it.